garrysmod/lua/includes/extensions/vector.lua

```diff
@@ -3,10 +3,257 @@ local meta = FindMetaTable( "Vector" )
 -- Nothing in here, still leaving this file here just in case
 
 --[[---------------------------------------------------------
-	Converts Vector To Color - alpha precision lost, must reset
+Converts Vector to Color - alpha precision lost, must reset
 -----------------------------------------------------------]]
-function meta:ToColor( )
+function meta:ToColor(num)
+	local a = (num == nil and 1 or num)

```

Comment:
This doesn't need to be a local - it can be placed inline in the Color call.

Reply:
Ok. Done.